pkg/client/networks.go

```diff
@@ -0,0 +1,163 @@
+package client
```

Comment:
This file is really just the batteries to include with the `Client` to make it easy for users to select which network and EVM backend to use.